CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+<a name="1.123.1"></a>
+## [1.123.1](https://github.com/mozilla/fxa-auth-db-mysql/compare/v1.122.1...v1.123.1) (2018-10-22)
+
+
+
+
+<a name="1.123.0"></a>
+# [1.123.0](https://github.com/mozilla/fxa-auth-db-mysql/compare/v1.121.0...v1.123.0) (2018-10-16)
```

Comment:
This changelog entry looks like a dup, not sure why but please remove it by hand before merge.